packages/web/src/dev/ValidatorsList.scss

```diff
@@ -0,0 +1,233 @@
+// TODO: Move to files
+// START // Colors
+$palette: (
```

Comment:
i'm not a fan of duplicating these color variables here.

Reply:
We could investigate a way to use the colors in SCSS and TypeScript, but in other PR.

Reply:
yes and why not keep with the norms of the project and use Stylesheet?  I'm definitely  not one to rally for consistency just for its own sake, and to me that a change in how styling is done/ new way of doing it should have a discussion around it as it could lead to an increasingly large bundle size if down the road more ways of styling are added and could create confusion for future developers. 🤷🏻‍♂️